js/jsonld.js

```diff
@@ -821,6 +821,8 @@ jsonld.normalize = function(input, options, callback) {
  * @param [options] the options to use:
  *          [format] the format if dataset param must first be parsed:
  *            'application/nquads' for N-Quads (default).
+ *          [useStandardNQuadsType] the format of N-Quads
+ *            (default: false)
```

Comment:
I don't think we need this option. All we're really doing is correcting for what could now be seen as a typo ... so if someone uses `application/nquads` for the `inputFormat`, let's just correct it for them and then proceed as if the proper format was given.